stingray/lightcurve.py

```diff
@@ -156,8 +156,9 @@ def __init__(self, time, counts, err=None, input_counts=True,
                 err_low, err_high = poisson_conf_interval(np.asarray(counts),
                     interval='frequentist-confidence', sigma=1)
                 # calculate approximately symmetric uncertainties
-                err = (np.absolute(err_low) + np.absolute(err_high) -
-                       2 * np.asarray(counts))/2.0
+                err_low -= np.asarray(counts)
```

Comment:
Because `np.absolute(err_low) - np.asarray(counts)` is not the same as `np.absolute(err_low - np.asarray(counts))`. Say `err_low=1` and `counts=5`. Then `np.absolute(1)-5 = -4`, and `np.absolute(1 - 5)=4`.